BUILD.md

```diff
@@ -53,6 +53,11 @@ The following example will download the needed dependencies and install the RAFT
 ./build.sh libraft --install
 ```
 
+The `--no-thrust` flag can be used to install the headers with minimal dependencies:
+```bash
+./build.sh libraft --install --no-thrust
```

Comment:
maybe `--no-deps`? instead of `--no-thrust` if this captures more dependencies than thrust